test.sh

```diff
@@ -30,64 +30,63 @@ function setup_osbs() {
   OSBS_CLIENT_REPO=${OSBS_CLIENT_REPO:-https://github.com/containerbuildsystem/osbs-client}
   OSBS_CLIENT_BRANCH=${OSBS_CLIENT_BRANCH:-master}
 
-  # Pull fedora images from registry.fedoraproject.org
-  if [[ $OS == "fedora" ]]; then
-    IMAGE="registry.fedoraproject.org/$IMAGE"
-  fi
```

Comment:
Yeah, sorry there was previous discussion about this:

@chmeliik
Can you clarify why pulling from the "official" repos is not desirable?

@ben-alkov
It's not *possible*; GH just fails.

Regardless even of that, that code has been skipped for months now - it's in the wrong place (which is why GH not pulling from registries other than docker.io hasn't been a problem up to this point).

@chmeliik
I see, makes sense (weird that it would fail though, huh)